code/game/objects/effects/anomalies.dm

```diff
@@ -419,4 +419,43 @@
 			if(EXPLODE_LIGHT)
 				SSexplosions.lowturf += T
 
+/obj/effect/anomaly/hallucination
+	name = "hallucination anomaly"
+	icon_state = "blueshatter2"
```

Comment:
is this a new icon? should it have one, or is just reusing one ok here

Reply:
should be good to use the existing icons, but i'll make a copy and name it properly, that way if someone changes that icon won't affect the anomaly